src/main/java/racingcar/controller/RaceGameController.java

```diff
@@ -0,0 +1,41 @@
+package racingcar.controller;
+
+import java.util.List;
+import racingcar.model.Car;
+import racingcar.model.Cars;
+import racingcar.model.RaceGame;
+import racingcar.view.InputView;
+import racingcar.view.OutputView;
+
+public class RaceGameController {
+
+    private final Cars cars;
+    private final RaceGame raceGame;
+
+
+    public RaceGameController(Cars cars, RaceGame raceGame) {
+        this.cars = cars;
+        this.raceGame = raceGame;
+
+    }
+
+    public void run() {
+
+        InputView.printEnterCarNames();
+        String enteredCarNames = InputView.getInput();
+
+        List<Car> racingCars = cars.extractValidCars(enteredCarNames);
```

Comment:
네이밍이나 줄바꿈을 적절히 잘 사용해주셔서, 지금도 너무 깔끔해서 잘 읽히지만, 의미가 비슷한 코드끼리 하나로 묶어서 controller의 메서드로 분리해서 run함수에 넣는 건 어떨까요? 

게임 전체의 흐름이 잘 보일 수 있을 거 같습니다 👍 

예를 들어서, '유효한 자동차 입력 받기' 역할이라고 가정하고, 메서드를 작성해봅니다!

```suggestion
private List<Car> getValidCarsFromInput() {
        InputView.printEnterCarNames();
        String enteredCarNames = InputView.getInput();
        return cars.extractValidCars(enteredCarNames);
    }

Reply:
위엣분 말처럼 각 인풋, 실행, 출력 이렇게 분리할 수 있을 거 같습니다!

Reply:
저도 메시지를 출력하고, 입력을 받는 부분이 코드 상으로는 두 줄이만 반복되어서 줄일 수 있는 방법이 없을까에 대한 고민을 했었는데, 해결책을 주셨네요! 제가 고민했던 방법은 inputView에서 메시지를 출력하고 입력까지 받는 방법이었는데 메서드가 한 가지 일만 하도록 하고 싶어서 그 방법은 선택 안 했었어요. 그런데 말씀해 주신대로 `getValidCarsFromInput()`과 같은 함수를 만들어 run에서 사용하면 모두 해결될 것 같네요! 답변 감사합니다